scopes/component/renaming/rename.cmd.ts

```diff
@@ -20,18 +20,18 @@ export class RenameCmd implements Command {
     },
     {
       name: 'new-name',
-      description: 'the new component name (without its scope name)',
+      description: "the new component name (without its scope name. use --scope to define the new component's scope)",
     },
   ];
   group = 'collaborate';
   skipWorkspace = true;
   alias = '';
   options = [
-    ['s', 'scope <scope-name>', 'default scope for the newly created component'],
+    ['s', 'scope <scope-name>', 'define the scope for the newly created component'],
     [
       'p',
       'path <relative-path>',
-      'relative path in the workspace. by default the path is `<scope>/<namespace>/<name>`',
+      'relative path in the workspace to place new component in. by default the path uses your workspace\'s "defaultScope" value',
```

Comment:
I think it's useful to mention the structure how the path is build: `<scope>/<namespace>/<name>`

Reply:
Dont think it should refer to a specific value for the defaultDirectory - it could be anything. Also, the pattern that was here is only if they stay with the default value from the workspace.jsonc's generation, they could have put anything there, and this will only confuse them.

Also, how to build a defaultDirectory is documented in the workspace.jsonc itself so no need to add it here in an unrelated command.

Reply:
Agree